sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/DownloadManifestResult.cs

```diff
@@ -5,7 +5,7 @@
 using System.IO;
 using System.Text.Json;
 
-namespace Azure.Containers.ContainerRegistry.Specialized
+namespace Azure.Containers.ContainerRegistry
 {
     /// <summary>
     /// The result from downloading an OCI manifest from the registry.
```

Comment:
nit and out of scope on this PR: wonder if we should rename `AsOciManifest` to `AsOciImageManifest` across the languages (so we can add `AsOciArtifactManifest` if/when artifact manifest comes)

Reply:
The feedback from Ted was that we should change the name of this method to make it clear it is deserializing the BinaryData Content property.